src/components/PokemonList.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+export default function PokemonList({ pokemons, gotoDetailsPage, addPokemonToCart}) {
+  return (
+    <div className="pokemon-list-container">
+      <table className="table">
+        <thead>
+          <tr>
+            <th scope="col">#</th>
+            <th scope="col">Name</th>
+            <th scope="col">Price</th>
+            <th scope="col">Actions</th>
+          </tr>
+        </thead>
+        <tbody>
+        {pokemons.map((pokemon, index) => (
+          <tr key={pokemon.name}>
```

Comment:
Better create a new component for this `<tr>`

Reply:
Sure, a component that describes a Pokemon :+1: , thanks